### DISCUSSION TITLE
Welcome to AI-TTS-Donations Discussions!

### DISCUSSION THREAD

**Discussion by mmattbtw** [mmattDonk/AI-TTS-Donations#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StinkyJutsu** [comment]
I belive this is the correct hub for solrock. I recently discovered you offered an ai extension that is free, and it is part of a fairly simple setup to create a stream pet with integrated ai.  it's a shame to see it is shutting down. i recently set up a locally ran ooga booga with a pygmalion model that has silero tts and whisper sst injected and thought that perhaps you could use that setup to continue running your solrock webpage. I have almost no experience with coding or javascript but i can see the potential in a locally ran text generation web UI on a dedicated server for a website like solrock. does that even make sense? lol
